rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -200,6 +209,41 @@ lv2_fs_mount_point* lv2_fs_object::get_mp(std::string_view filename)
 	return &g_mp_sys_dev_root;
 }
 
+std::string lv2_fs_object::get_vfs(std::string_view filename)
+{
+	const auto mp_name = get_device_path(filename);
+
+	if (!mp_name.empty())
+	{
+		if (mp_name == "dev_hdd0"sv)
+			return g_cfg_vfs.get(g_cfg_vfs.dev_hdd0, rpcs3::utils::get_emu_dir());
+		if (mp_name == "dev_hdd1"sv)
+			return g_cfg_vfs.get(g_cfg_vfs.dev_hdd1, rpcs3::utils::get_emu_dir());
+		if (mp_name.starts_with("dev_usb"sv))
+			return g_cfg_vfs.get_device(g_cfg_vfs.dev_usb, std::string(mp_name).insert(0, 1, '/'), rpcs3::utils::get_emu_dir()).path;
```

Comment:
This probably reads more intuitively
```suggestion
			return g_cfg_vfs.get_device(g_cfg_vfs.dev_usb, fmt::format("/%s", mp_name), rpcs3::utils::get_emu_dir()).path;
```

Reply:
Done.